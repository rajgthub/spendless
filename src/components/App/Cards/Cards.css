.container {
  display: grid;
  grid-gap: 10px;
  grid-template-columns: repeat(auto-fit, minmax(450px, min-content));
  justify-content: center;
  /* background-color: rgba(224, 97, 97, 0.164); */
  color: #444;
  margin: 1rem;
}
.title {
  background-color: #00596b;
  text-align: center;
  text-transform: uppercase;
  color: white;
}
.card_left,
.card_right {
  border-radius: 5px;
  border: 2px solid #72a1aa;
  display: flex;
  flex-direction: column;
  justify-content: space-between;
}
.card_body {
  padding: 1rem;
}
@media only screen and (max-width: 700px) {
  .container {
    display: grid;
    grid-gap: 10px;
    grid-template-columns: repeat(auto-fit, minmax(380px, min-content));
    margin: .5rem;
  }
 html{
   font-size: 52.2%;
 }
}
