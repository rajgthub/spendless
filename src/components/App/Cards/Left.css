.heading{
    color: rgb(70, 207, 132);
    font-size: 2.5rem;
    font-weight: bold;
}