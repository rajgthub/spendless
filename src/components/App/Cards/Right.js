import React from 'react'
import SpendLessFilter from '../SpendLessFilter/index';
 const Right = () => (
     <div className="card_right">
     <SpendLessFilter />   
     </div>
 )
export default Right
