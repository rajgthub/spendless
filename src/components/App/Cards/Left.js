import React from "react";
import "./Left.css";
import AnualIncomes from "../AnualIncomes";
import MonthlySpending from "../MonthlySpending";
const Left = () => (
  <div className="card_left">
    <div className="title">your income & spend</div>
    <section className="card_body">
      <h1 className="heading">Anual incomes</h1>
      <AnualIncomes />
      <h1 className="heading">Monthly spending</h1>
      <MonthlySpending />
    </section>
  </div>
);
export default Left;
