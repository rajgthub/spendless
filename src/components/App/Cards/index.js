import React from 'react'
import Left from './Left';
import Right from './Right';
import './Cards.css'
const Cards = () => {
  return (
    <div className="container">
      <Left />
      <Right />
    </div>
  )
}
export default Cards