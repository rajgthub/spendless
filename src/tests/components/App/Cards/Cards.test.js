import React from "react";
import ReactShallowRenderer from "react-test-renderer/shallow";
import Cards from '../../../../components/App/Cards/index';
test("should render Header ", () => {
  const renderer = new ReactShallowRenderer();
  renderer.render(<Cards />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
});
