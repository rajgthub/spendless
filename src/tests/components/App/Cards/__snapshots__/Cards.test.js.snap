// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Header  1`] = `
<div
  className="container"
>
  <Left />
  <Right />
</div>
`;
